1-tolean.py

```diff
@@ -493,12 +494,26 @@ def check_opt(opt, timeout, bitwidth, hide_progress):
 
   return True # succeeded, did not time out
 
+def sanitize_name(name):
+  renamed = name.replace(':', '_').replace('-','_').replace(' ','_')
+  #TODO: this should be a regex but I don't know the python syntax for it ATM
+  for s in ["(",")", "~", "&", ">"]:
+    pos = renamed.find(s)
+    if pos != -1:
+      renamed = renamed[:pos]
+  return renamed
```

Comment:
```suggestion
import re

def sanitize_name(name):
    renamed = re.sub(r'[()~&>]', '', name)
    renamed = re.sub(r'[:, -]', '_', renamed)
    return renamed
```

Reply:
thanks! I feel like I should explain that I wrote thin on the plane and couldn't look up the syntax, hence the TODO 😅